articles/digital-twins/concepts-models.md

```diff
@@ -212,7 +212,7 @@ The following example re-imagines the Home model from the earlier DTDL example a
 
 In this case, Core contributes an ID and name to Home. Other models can also extend the Core model to get these properties as well. Here's a Room model extending the same parent interface:
 
-:::code language="json" source="~/digital-twins-docs-samples-getting-started/models/advanced-home-example/IRoom.json" range="2-9" highlight="6":::
+:::code language="json" source="~/digital-twins-docs-samples-getting-started/models/advanced-home-example/IRoom.json" range="2-9" highlight="10":::
```

Comment:
Requested change: `range="2-10" highlight="9""`

Explanation: Since the line that needs to be highlighted is now line 10 of the file, the range should be extended to include line 10. Since the display range starts at line 2, the highlight count also starts at line 2, making the line that needs to be highlighted line 9 of the displayed snippet.
![image](https://github.com/user-attachments/assets/65de097b-4b9b-42e0-b12d-2f0260912b16)